src/frontends/tests/onnx/onnx_import.in.cpp

```diff
@@ -3429,6 +3429,84 @@ NGRAPH_TEST(${BACKEND_NAME}, onnx_roi_align_f32) {
+         18.7,  19.8,  20.9,  22.,   23.1,  24.2,  25.3,  26.4,  27.5,  28.6,  29.7,  30.8,  31.9,  33.,   34.1,  35.2,
+         36.3,  37.4,  38.5,  39.6,  40.7,  41.8,  42.9,  44.,   45.1,  46.2,  47.3,  48.4,  49.5,  50.6,  51.7,  52.8,
+         53.9,  55.,   56.1,  57.2,  58.3,  59.4,  60.5,  61.6,  62.7,  63.8,  64.9,  66.,   67.1,  68.2,  69.3,  70.4,
+         71.5,  72.6,  73.7,  74.8,  75.9,  77.,   78.1,  79.2,  80.3,  81.4,  82.5,  83.6,  84.7,  85.8,  86.9,  88.,
+         89.1,  90.2,  91.3,  92.4,  93.5,  94.6,  95.7,  96.8,  97.9,  99.,   100.1, 101.2, 102.3, 103.4, 104.5, 105.6,
+         106.7, 107.8, 108.9, 110.,  111.1, 112.2, 113.3, 114.4, 115.5, 116.6, 117.7, 118.8, 119.9, 121.,  122.1, 123.2,
+         124.3, 125.4, 126.5, 127.6, 128.7, 129.8, 130.9, 132.,  133.1, 134.2, 135.3, 136.4, 137.5, 138.6, 139.7, 140.8,
+         141.9, 143.,  144.1, 145.2, 146.3, 147.4, 148.5, 149.6, 150.7, 151.8, 152.9, 154.,  155.1, 156.2, 157.3, 158.4,
+         159.5, 160.6, 161.7, 162.8, 163.9, 165.,  166.1, 167.2, 168.3, 169.4, 170.5, 171.6, 172.7, 173.8, 174.9, 176.,
+         177.1, 178.2, 179.3, 180.4, 181.5, 182.6, 183.7, 184.8, 185.9, 187.,  188.1, 189.2, 190.3, 191.4, 192.5, 193.6,
+         194.7, 195.8, 196.9, 198.,  199.1, 200.2, 201.3, 202.4, 203.5, 204.6, 205.7, 206.8, 207.9, 209.,  210.1, 211.2,
+         212.3, 213.4, 214.5, 215.6, 216.7, 217.8, 218.9, 220.,  221.1, 222.2, 223.3, 224.4, 225.5, 226.6, 227.7, 228.8,
+         229.9, 231.,  232.1, 233.2, 234.3, 235.4, 236.5, 237.6});
+
+    test_case.add_input<float>({0, 0, 0.75, 2.2, 1.2, 0.5, 2.8, 1.9, 0, 3, 0, 3});
+
+    test_case.add_input<int64_t>({0, 2, 1});
+    test_case.add_expected_output<float>(
+        Shape{3, 2, 4, 4},
+        {1.1,       1.1,       1.1,       1.1,       1.1,       1.1,       1.1,       1.1,       2.3375,    2.3375,
+         2.3375,    2.3375,    4.1525,    4.1525,    4.1525,    4.1525,    40.7,      40.7,      40.7,      40.7,
+         40.7,      40.7,      40.7,      40.7,      41.9375,   41.9375,   41.9375,   41.9375,   43.7525,   43.7525,
+         43.7525,   43.7525,   159.72,    159.94,    160.16,    160.38,    159.90562, 160.12563, 160.34563, 160.56563,
+         160.9575,  161.1775,  161.3975,  161.61751, 162.1125,  162.3325,  162.55249, 162.77249, 199.32,    199.54001,
+         199.76001, 199.97998, 199.50562, 199.72563, 199.94562, 200.16562, 200.5575,  200.7775,  200.9975,  201.2175,
+         201.7125,  201.93251, 202.1525,  202.37251, 86.9,      86.9,      86.9,      86.9,      86.9,      86.9,
+         86.9,      86.9,      86.9,      86.9,      86.9,      86.9,      86.9,      86.9,      86.9,      86.9,
+         126.5,     126.5,     126.5,     126.5,     126.5,     126.5,     126.5,     126.5,     126.5,     126.5,
+         126.5,     126.5,     126.5,     126.5,     126.5,     126.5});
+    test_case.run_with_tolerance_as_fp(0.01f);
```

Comment:
This tolerance is a little sketchy, is the error that high acceptable in this case?

Reply:
the template_plugin use “0.01f” as its default threshold for its testcases, so I think if none special request for this testcase, it seems ok here.